lib/controller.js

```diff
@@ -651,9 +651,30 @@ controller.getWifiInfo = function(opts) {
 
 controller.connectToNetwork = function(opts) {
   opts.authorized = true;
-  return controller.standardTesselCommand(opts, function(tessel) {
-    return tessel.connectToNetwork(opts);
-  });
+  var ssid = opts.ssid;
+  var password = opts.password;
+  var security = opts.security;
+  var securityOptions = ['none', 'wep', 'psk', 'psk2', 'wpa', 'wpa2'];
```

Comment:
We may want to make these options a property of the Tessel class (in the chance that we ever use this CLI with other hardware).